### PR TITLE
MAINT: employ run_doctests, closes #256

### LINKED ISSUE #256: Utility to run doctests
Besides pylint warnings, the lines in each file responsible for doctest add some redundancy. This should be reduced by making a small `run_doctests()` utility.

### PR DESCRIPTION
**Pull request by kohr-h**

### CODE REVIEW

**Comment by kohr-h** [comment]
Checks pass.

File: odl/discr/discr_mappings.py [hunk 1]
```diff
@@ -23,20 +23,18 @@
...
-# External imports
 from itertools import product
 import numpy as np
 
-# ODL imports
 from odl.operator.operator import Operator
 from odl.discr.partition import RectPartition
 from odl.space.base_ntuples import NtuplesBase, FnBase
 from odl.space.fspace import FunctionSet, FunctionSpace
+from odl.util.testutils import run_doctests
```
Comment (adler-j) [inline odl/discr/discr_mappings.py]: How about put this inside the "if main"?

**Comment by adler-j** [comment]
As mentioned above I feel like this import should go into the if main thing, else I'm fine.

File: odl/discr/discr_mappings.py (hunk 1 above)
Comment (kohr-h) [inline odl/discr/discr_mappings.py]: That's where all the pylint complaints come from :-)

Now we don't have to obey a linter, and I agree that the additional import is somewhat clumsy. I'll move the import.

Comment (adler-j) [inline odl/discr/discr_mappings.py]: Perhaps add an ignore to this line then for pylint?

Comment (kohr-h) [inline odl/discr/discr_mappings.py]: Yeah, that one place is acceptable.

**Pull request merged**